### PR TITLE
feat: decreased UR density

### PR DESCRIPTION
**Pull request by xardass**
## **Description**
This PR modifies the density of the animated QR codes so that they can be scanned by the ZERO hardware wallet from NGRAVE. 

## **Related issues**

Fixes: NA

## **Manual testing steps**

1. Connect a QR hardware
2. Send a Transaction (eg. using the test app at https://metamask.github.io/test-dapp/) 
3. See the QR code

### **Before**

https://github.com/MetaMask/metamask-extension/assets/26218337/9d1c6a6c-c345-4206-b6d0-cd8329817114

### **After**

https://github.com/MetaMask/metamask-extension/assets/26218337/1393ea86-494c-48d5-83fd-ea801325709e

## **Pre-merge author checklist**

- [x] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/CODING_GUIDELINES.md).
- [x] I've clearly explained what problem this PR is solving and how it is solved.
- [x] I've linked related issues
- [x] I've included manual testing steps
- [x] I've included screenshots/recordings if applicable
- [x] I’ve included tests if applicable
- [x] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [x] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.
- [x] I’ve properly set the pull request status:
  - [ ] In case it's not yet "ready for review", I've set it to "draft".
  - [x] In case it's "ready for review", I've changed it from "draft" to "non-draft".

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

**Comment by xardass** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by gantunesr — DISMISSED** [review]

**Review by FrederikBolding — DISMISSED** [review]
Code LGTM, but should be QAed!

**Comment by FrederikBolding** [comment]
I can't rebase this PR, so you will have to do that @xardass to get it passing. Some of the E2E tests were failing on forks but that is fixed on the latest `develop`.

**Review by danroc — DISMISSED** [review]
LGTM. I tested with `personal_sign` using a different QR code based wallet.

**Comment by danroc** [comment]
@xardass, is 200 the largest fragment size that the ZERO can support? If so, you should add a comment explaining your change. Otherwise, it might inadvertently be reverted in the future.

**Review by montelaidev — DISMISSED** [review]

**Comment by gantunesr** [comment]
@vivek-consensys will support with QA

**Comment by vivek-consensys** [comment]
Please see screen recording of test flows:- 

- Connect QR Hardware Wallet (Keystone)
- Send legacy tx (using test-dapp)
- Eth sign (using test-dapp) _had a technical issue when testing this, so took a bit longer than expected_
- Signed typed data v4 (using test-dapp)

https://github.com/MetaMask/metamask-extension/assets/106310394/5be51066-1c0c-41db-8568-2e3b010f5b62

**Review by FrederikBolding — DISMISSED** [review]

File: ui/components/app/qr-hardware-popover/qr-hardware-sign-request/player.js [hunk 1]
```diff
@@ -15,7 +15,8 @@ import { Text, Box } from '../../../component-library';
 const Player = ({ type, cbor, cancelQRHardwareSignRequest, toRead }) => {
   const t = useI18nContext();
   const urEncoder = useMemo(
-    () => new UREncoder(new UR(Buffer.from(cbor, 'hex'), type), 400),
+    //For NGRAVE ZERO support please keep to a maximum fragment size of 200
```
Comment (FrederikBolding) [inline ui/components/app/qr-hardware-popover/qr-hardware-sign-request/player.js:18]:
```suggestion
    // For NGRAVE ZERO support please keep to a maximum fragment size of 200
```
To satisfy the linter!

**Review by FrederikBolding — APPROVED** [review]

**Comment by xardass** [comment]
I created the same PR for Metamask-Mobile: https://github.com/MetaMask/metamask-mobile/pull/8300

**Review by gantunesr — APPROVED** [review]

**Pull request merged**